core/src/main/java/dev/hypera/scaffolding/schematic/LegacyLookup.java

```diff
@@ -22,43 +22,55 @@
  */
 package dev.hypera.scaffolding.schematic;
 
-import com.google.gson.JsonArray;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
+import com.google.gson.*;
+import com.google.gson.stream.JsonReader;
 import dev.hypera.scaffolding.Scaffolding;
-import dev.hypera.scaffolding.region.Region;
-import java.io.BufferedReader;
-import java.io.IOException;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMaps;
+import it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap;
+import it.unimi.dsi.fastutil.io.FastBufferedInputStream;
+import org.jetbrains.annotations.ApiStatus.Internal;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
```

Comment:
`*` imports should not be used, `java.` imports should not be separate from the other imports.